### DISCUSSION TITLE
Display currency conversion

### DISCUSSION THREAD

**Discussion by GinoPane** [firefly-iii/firefly-iii#4683] [General]
Hi!
Is it possible/planned/doable to display reports with multi-currency transactions/accounts/etc in one chosen currency?
For example, we have a report which displays one transaction in Pounds and one transaction in Euros. It would be nice to choose, for example, USD and see both transactions in USD (or any other chosen currency).

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/#automated-currency-conversion-so-everything-is-in-euros-or-dollars

**Reply by GinoPane** [reply]
So the question is, why not? If the PR would be available for this, would you accept it? The purpose is displaying of results only. Even my bank applications do this.

**Reply by JC5** [reply]
Because it's a lot of work to add a reliable currency converter. It needs a working internet connection or another configuration interface to setup the rates. The result will always be inaccurate because banks set their own rates in their favor.

I'm thinking of a reliable way to add this, but don't hold your breath.

**Reply by Sleepful** [reply]
Not holding my breath or anything, but the only reliable way to add this would be for people to add their own code to fetch their bank's API (or another one that would be close enough).

**Reply by JC5** [reply]
Although currently unused, Firefly III contains a reliable currency converter that can download new rates from a curated GitHub repository. I hope to use it in the new v2 layout that is currently in development.
